src/SettingsMigrator/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine
+{
+    using System;
+    using System.Globalization;
+    using System.IO;
+    using CommandLineResources = Microsoft.VisualStudio.TestPlatform.CommandLine.Resources.Resources;
+
+    /// <summary>
+    /// Entry point for SettingsMigrator.
+    /// </summary>
+    class Program
```

Comment:
Please make it Public, if it doesn't contain code that needs to be tested, make it static as well

Reply:
Done